2_process.yml

```diff
@@ -22,7 +22,11 @@ sources:
   - 2_process/src/process_precip.R
   - 2_process/src/extract_storm_points.R
   - 2_process/src/interpolate_storm_points.R
+<<<<<<< HEAD
   - 2_process/src/process_river_geoms.R
+=======
+  - 2_process/src/normalize_streamdata.R
+>>>>>>> c7db744994199e0df09616abf49f80c3bb691d6b
```

Comment:
looks like we could keep both of these code lines, delete the git stuff

Reply:
woops. missed that.